client/app/scripts/superdesk-search/views/item-search.html

```diff
@@ -141,6 +141,6 @@
     </fieldset>    
 </form>
 <div class="actions">
-    <button class="btn btn-info pull-right" ng-click="search()" translate>Go</button>
+    <button class="btn btn-info pull-right" ng-click="search()" ng-disabled="!isSearchEnabled()" translate>Go</button>
```

Comment:
Well done, we need more refactorings like this (moving the more complex logic out of the templates).